packages/pluggableWidgets/rich-text-web/src/RichText.editorPreview.tsx

```diff
@@ -1 +1,14 @@
-export const preview = (): null => null;
+import { createElement, ReactElement } from "react";
+import RichTextPreviewSVGLight from "./assets/rich-text-preview-light.svg";
+import { RichTextPreviewProps } from "../typings/RichTextProps";
+
+export const preview = (props: RichTextPreviewProps): ReactElement => {
+    let doc = decodeURIComponent(RichTextPreviewSVGLight.replace("data:image/svg+xml,", "")).replace(
+        "width='602' height='148'",
+        ""
+    );
+
+    doc = props.stringAttribute ? doc.replace("[No attribute selected]", `[${props.stringAttribute}]`) : doc;
+
+    return <div className="widget-rich-text" style={{ maxWidth: 602 }} dangerouslySetInnerHTML={{ __html: doc }}></div>;
```

Comment:
Why we need it? `dangerouslySetInnerHTML` is not a good solution. Also the inline styles does not comply with CSP.